uefidir/image/startup.nsh

```diff
@@ -0,0 +1 @@
+FS0:\EFI\BOOT\BOOTA64.EFI
```

Comment:
Why is startup.nsh necessary here? It should autoboot EFI\BOOT\BOOTA64.EFI.

Reply:
It doesn't autoboot it for me though, and I'm not sure why. Are my efi vars just wrong?

Reply:
Very likely, but I am not sure how or why. I had the same issue with x86 uefi and it somehow resolved with a totally unrelated change

Reply:
Can confirm, fixing my boot settings makes it work.